manifests/resource/geo.pp

```diff
@@ -63,6 +63,10 @@
   Optional[Boolean] $proxy_recursive  = undef
 ) {
 
+  if ! defined(Class['nginx']) {
```

Comment:
I mostly just don't like `if !` vs `if not` but I don't have a strong opinion, and it's used all throughout the code, so I don't see any reason to change it.